api/src/test/java/marquez/SparkAgentIntegrationTest.java

```diff
@@ -0,0 +1,51 @@
+package marquez;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.fail;
+
+import java.io.IOException;
+import java.net.http.HttpResponse;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.List;
+import java.util.concurrent.CompletableFuture;
+import java.util.stream.Collectors;
+import org.junit.Test;
+import org.junit.experimental.categories.Category;
+import org.junit.runner.RunWith;
+import org.junit.runners.Parameterized;
+import org.junit.runners.Parameterized.Parameter;
+import org.junit.runners.Parameterized.Parameters;
+
+@RunWith(Parameterized.class)
+@Category(IntegrationTests.class)
+public class SparkAgentIntegrationTest extends BaseIntegrationTest {
```

Comment:
_Minor_: I feel the Marquez integration tests for spark would be more appropriate to have in the spark agent test suite which would allow us to 1) run a spark job in local mode 2) collect an open lineage event 3) query the Marquez to validate the metadata collected for the spark job (similar to our [approach with Airflow](https://github.com/MarquezProject/marquez/blob/main/integrations/airflow/tests/integration/integration.py)). But, make sense to have the integration test be part of the Marquez test suite for now 👍

Reply:
I'm not sure if its possible to test using source code rather than rebuilt agent/marquez code using any other method than this due to dependencies on the build process, jvm versions (not java versions), and other library dependency conflicts.

Reply:
Yeah, you might be right here. When we promote the spark integration status from prototype we can revisit.